.github/actions/deploy-portal/action.yml

```diff
@@ -45,6 +45,9 @@ inputs:
   NEXT_PUBLIC_FEATURE_FLAG_ENABLE_MAINNET:
     required: true
     description: Feature flag to enable Mainnet
+  NEXT_PUBLIC_FEATURE_FLAG_ENABLE_STAKE_CAMPAIGN:
```

Comment:
The proper name seems to be "staking campaign". Double check with @dvnahuel and the marketing team. If that is the case, everything should be "staking", not "stake".